src/Sylius/Behat/Page/Admin/Payment/IndexPage.php

```diff
@@ -33,9 +33,9 @@ public function getPaymentStateByOrderNumber(string $orderNumber): string
         return $this->getField($orderNumber, 'state')->getText();
     }
 
-    public function showOrderPageForNthPayment(int $position): void
+    public function showOrderPage(string $orderId): void
```

Comment:
I do not understand why are you changing this method so much. In my opinion in this step definition, we should be still finding a row with specific payment, finding by order ID is not a good solution as there can be many payments associated with one order :)

Reply:
You a right, I will fix it